airflow/decorators/__init__.pyi

```diff
@@ -126,7 +126,22 @@ class TaskDecoratorCollection:
             such as transmission a large amount of XCom to TaskAPI.
         """
     @overload
-    def virtualenv(self, python_callable: Function) -> Function: ...
+    def virtualenv(self, python_callable: F) -> F: ...
```

Comment:
Why is this line changed? There is no `F`.

Reply:
Reverted change.